package.json

```diff
@@ -1,25 +1,20 @@
 {
-  "name": "llama-tokenizer-js",
-  "version": "1.1.3",
-  "description": "JS tokenizer for LLaMA-based LLMs",
-  "main": "llama-tokenizer.js",
+  "name": "llama-tokenizer",
+  "private": true,
```

Comment:
private?

Reply:
same reason as this one is for monorepo config not for the package.